UPGRADE.md

```diff
@@ -100,3 +100,82 @@
+| sylius_invoicing_plugin.generator.invoice_number_generator                         | sylius_invoicing.generator.invoice_number_generator                         |
+| sylius_invoicing_plugin.generator.invoice_identifier                               | sylius_invoicing.generator.invoice_identifier                               |
+| sylius_invoicing_plugin.generator.invoice                                          | sylius_invoicing.generator.invoice                                          |
+| sylius_invoicing_plugin.generator.invoice_pdf_file                                 | sylius_invoicing.generator.invoice_pdf_file                                 |
+| sylius_invoicing_plugin.creator.invoice                                            | sylius_invoicing.creator.invoice                                            |
+| sylius_invoicing_plugin.creator.mass_invoices                                      | sylius_invoicing.creator.mass_invoices                                      |
+| sylius_invoicing_plugin.generator.pdf_options                                      | sylius_invoicing.generator.pdf_options                                      |
+| sylius_invoicing_plugin.generator.twig_to_pdf                                      | sylius_invoicing.generator.twig_to_pdf                                      |
+| sylius_invoicing_plugin.event_producer.order_payment_paid                          | sylius_invoicing.event_producer.order_payment_paid                          |
+| sylius_invoicing_plugin.event_listener.order_placed                                | sylius_invoicing.event_listener.order_placed                                |
+| sylius_invoicing_plugin.event_producer.order_placed                                | sylius_invoicing.event_producer.order_placed                                |
+| sylius_invoicing_plugin.listener.order_payment_paid                                | sylius_invoicing.event_listener.order_payment_paid                          |
+| sylius_invoicing_plugin.ui.menu.admin_menu_listener                                | sylius_invoicing.ui.menu.admin_menu_listener                                |
+| sylius_invoicing_plugin.email.invoice_email_sender                                 | sylius_invoicing.email.invoice_email_sender                                 |
+| sylius_invoicing_plugin.command_handler.send_invoice_email                         | sylius_invoicing.command_handler.send_invoice_email                         |
+| sylius_invoicing_plugin.provider.tax_rate_percentage                               | sylius_invoicing.provider.tax_rate_percentage                               |
+| sylius_invoicing_plugin.custom_factory.invoice                                     | sylius_invoicing.custom_factory.invoice                                     |
+| sylius_invoicing_plugin_security.voter.invoice                                     | sylius_invoicing.security.voter.invoice                                     |
```

Comment:
Missing pretty much all service ids that were previously FQCNs, also add all of the resource-generated services as well, just so we have one list with everything.